tests/async_tests/test_interfaces.py

```diff
@@ -212,6 +212,25 @@ async def test_http_proxy(proxy_server: URL, proxy_mode: str) -> None:
         assert reason == b"OK"
 
 
+@pytest.mark.parametrize("local_address", [b"0.0.0.0"])
+@pytest.mark.asyncio
+# This doesn't run with trio, since trio doesn't support local_address.
+async def test_http_request_local_address(local_address: str) -> None:
+    async with httpcore.AsyncConnectionPool(local_address=local_address) as http:
```

Comment:
Possible simplification?

```suggestion
@pytest.mark.asyncio
# This doesn't run with trio, since trio doesn't support local_address.
async def test_http_request_local_address() -> None:
    async with httpcore.AsyncConnectionPool(local_address="0.0.0.0") as http:
```